### DISCUSSION TITLE
How to verify if checkbox is checked

### DISCUSSION THREAD

**Discussion by Muhammadehsan2019** [cypress-io/cypress#15013] [Questions and Help] [answered]
how can I verify if the checkbox is checked. I want to use the below scenario:
if (element.isChecked){
   // if checked then click to uncheck and then verify its unchecked
    element.click()
    expect(element).not.to.be.checked
}

I used jquerry method with folwoing methods, but nothing worked for me 
 each(function), and then(function(func)=>{
..............................
..............................
...............
})

**Comment by bahmutov** [comment, marked as answer]
In general, this is an anti-pattern https://on.cypress.io/conditional-testing

But if you must do it, see https://glebbahmutov.com/cypress-examples/6.4.0/recipes/conditional-testing.html
